.github/workflows/model-monitoring-ci.yml

```diff
@@ -0,0 +1,88 @@
+name: model-monitoring-ci
+
+on:
+  pull_request:
+    branches:
+      - main 
+    paths:
+      - model_monitoring/**
+      - .github/workflows/model-monitoring-ci.yml
+      
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+env:
+  root_dir: root
+  momoComponentsRootPath: root/model_monitoring/components
+  testsRootPath: root/model_monitoring/components/tests
+  pytest_report_folder: results
+  pytest_report_file: junit3.xml
+
+permissions:
+  # Required to clone repo
+  contents: read
+  # Required for OIDC login to Azure
+  id-token: write
+
+defaults:
+  run:
+    shell: cmd
+
+jobs:
+  run-e2e-tests:
+    name: Run Monitoring Component Tests
+    runs-on: windows-latest
```

Comment:
Does it specifically need Windows? This will be the first non-Linux test in the repo, I believe.

Reply:
let me see if ubuntu works! It used to need windows due to the azure cli task in ADO :)

Reply:
Looks like a lot of the commands are still windows-specific - I'll do a follow-up to convert to ubuntu once the e2e is in :)

Reply:
Sounds good. Thanks for testing.